### PR TITLE
[v7r0] Production system: docs and system test

### PR DESCRIPTION
**Pull request by arrabito**
BEGINRELEASENOTES

*docs
NEW: added Production system documentation

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
Please run autopep8 on your code (see https://travis-ci.org/DIRACGrid/DIRAC/jobs/559310430).

e.g. `autopep8 --global-config setup.cfg --in-place your/file/.py`

File: ProductionSystem/DB/ProductionDB.py [hunk 1]
```diff
@@ -267,6 +257,7 @@ def startProduction(self, prodName, connection=False):
     if not res['OK']:
       return res
 
+    print res['Value']
```
Comment (atsareg) [inline ProductionSystem/DB/ProductionDB.py]: I guess this print is a debugging artifact ?

**Comment by atsareg** [comment]
I merge it now and I will remove the offending print statement

**Comment by arrabito** [comment]
OK. I will commit tomorrow a few other fixes.

**Pull request merged**